src/api/lib/tasks/delayed_job.rake

```diff
@@ -20,8 +20,7 @@ task(importrequests: :environment) do
       if r.save
         Rails.logger.info "Request ##{lastrq} imported"
       else
-        Rails.logger.error "Request ##{lastrq} could not be saved:\n%s" \
-                           % r.errors.full_messages.join("\n")
+        Rails.logger.error format("Request ##{lastrq} could not be saved:\n%<error>s", error: r.errors.full_messages.join("\n"))
```

Comment:
Also nicer (I'm a ruby noob for the most part). I'd personally continue the `error` argument on the next line, but it seems like long lines are acceptable here.

Reply:
We use [Rubocop](https://github.com/rubocop-hq/rubocop) to decide it for us :)

Reply:
I figured, but you did change the limit from the [default of 80](https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml#L1681) to [150](https://github.com/openSUSE/open-build-service/blob/master/.rubocop.yml#L55).

I guess I need to widen my windows when working on OBS :smile:

Reply:
Well, if you don't want to, you can still run ```rake dev:lint:rubocop:auto_correct``` and it will fix it for you :)

Reply:
Interesting. Good to know!